### ISSUE TITLE
Animated AVIF File Not Playing

### ISSUE THREAD

**Issue by Jay123210599** [d2phap/ImageGlass#2033]
### System information
- Windows OS version: Windows 11
- ImageGlass version: 9.1.8.723
- ImageGlass release: Classic

### To Reproduce

I have a lossless animated AVIF file, and I want to play it on ImageGlass.

### Actual behavior

It just shows it as a still image.

### Expected behavior

It should play all frames from the animated AVIF.

### Screenshots / Video / Sample image file

![Screenshot (1420)](https://github.com/user-attachments/assets/a881be92-0c29-4a24-a94e-4b607cecbeb5)

Sample file: https://drive.google.com/file/d/1HaW8ZUCKlcN6-skcs3F4V8-E2VIfc2jR/view?usp=sharing

### Additional context

Does ImageGlass support playing animated AVIF files? Is there a version of it that does?

**Comment by dmak** [comment]
@Jay123210599 Would be great if you mention the way you have created that file so one can reproduce those steps.

Do you know any viewer that displays that file correctly? I am personally interested in image viewer that can display multi-image AVIF, and it seems that ImageGlass always displays the 1st image in the sequence.

**Comment by Jay123210599** [comment]
> @Jay123210599 Would be great if you mention the way you have created that file so one can reproduce those steps.
> 
> Do you know any viewer that displays that file correctly? I am personally interested in image viewer that can display multi-image AVIF, and it seems that ImageGlass always displays the 1st image in the sequence.

I made that file from a video with this fffmpeg command:
`ffmpeg -i  -c:v libaom-av1 -crf 0 -cpu-used 4 output.avif` 

I tried XnViewMP and Irfanview and neither of them played the avif correctly.

**Comment by gsxgmpkt9020** [comment]
> > [@Jay123210599](https://github.com/Jay123210599) Would be great if you mention the way you have created that file so one can reproduce those steps.
> > Do you know any viewer that displays that file correctly? I am personally interested in image viewer that can display multi-image AVIF, and it seems that ImageGlass always displays the 1st image in the sequence.
> 
> I made that file from a video with this fffmpeg command: `ffmpeg -i -c:v libaom-av1 -crf 0 -cpu-used 4 output.avif`
> 
> I tried XnViewMP and Irfanview and neither of them played the avif correctly.

Use JarkViewer instead to play it

**Comment by dmak** [comment]
> Use JarkViewer instead to play it

I just tried [JarkViewer](https://github.com/jark006/JarkViewer/releases) and I see that it flips all images in AVIF in endless loop (and `ffmpeg ... -loop 0 output.avif` does not help). At least it can "see" all images in sequence, but there is no manual navigation (previous/next image).

**Comment by gsxgmpkt9020** [comment]
You should follow the keys to operate:

![Image](https://github.com/user-attachments/assets/e380136e-61c1-4479-958c-27cf9bd01ae6)



Or try this old school app to play with.
![Image](https://github.com/user-attachments/assets/dee5ceca-7cf2-4ccd-b246-a7f01ca12ea3)

https://www.nyam.pe.kr/dev/imagine/

If file is large , then should wait for seconds to load

**Comment by dmak** [comment]
> You should follow the keys to operate:
> 
> Or try this old school app to play with. ![Image](https://github.com/user-attachments/assets/dee5ceca-7cf2-4ccd-b246-a7f01ca12ea3)
> 
> https://www.nyam.pe.kr/dev/imagine/

Could you please share the steps you have used to create multi-image AVIF? For the file I have created, `ffmpeg` reports:
```
Stream #0:1[0x1](und): Video: av1 (libdav1d) (Main) (av01 / 0x31307661), yuv444p(tv, bt470bg/unknown/unknown), 800x800
 [SAR 1:1 DAR 1:1], 146 kb/s, 1 fps, 1 tbr, 16384 tbn (default)
```
and that AVIF file does not play with [Imagine](https://www.nyam.pe.kr/dev/imagine/) 😭

**Comment by gsxgmpkt9020** [comment]
> > You should follow the keys to operate:
> > Or try this old school app to play with. ![Image](https://github.com/user-attachments/assets/dee5ceca-7cf2-4ccd-b246-a7f01ca12ea3)
> > https://www.nyam.pe.kr/dev/imagine/
> 
> Could you please share the steps you have used to create multi-image AVIF? For the file I have created, `ffmpeg` reports:
> 
> ```
> Stream #0:1[0x1](und): Video: av1 (libdav1d) (Main) (av01 / 0x31307661), yuv444p(tv, bt470bg/unknown/unknown), 800x800
>  [SAR 1:1 DAR 1:1], 146 kb/s, 1 fps, 1 tbr, 16384 tbn (default)
> ```
> 
> and that AVIF file does not play with [Imagine](https://www.nyam.pe.kr/dev/imagine/) 😭

I asked Google gemini and so on. Ai will tell you.

Below is some examples of mine.

```
ffmpeg -framerate 30 -start_number 0 -i 'C:\sort\1\webp_%05d.png' ` -c:v libsvtav1 -preset 5 -crf 18 -pix_fmt yuv420p ` -vf 'scale=trunc(iw/2)*2:trunc(ih/2)*2' -an -f ivf output.ivf
.\MP4Box.exe -add-image video.ivf:id=1:primary -ab avif -ab miaf -brand avis -add video.ivf:hdlr=pict:ccst:name= -new final.avif
```

**Comment by vanontom** [comment]
I'd also love to see animated AVIF support in  ImageGlass!

I'm using FFmpeg (svt-av1) to create short looping animations from videos (without quality loss vs ancient GIF). Surprisingly Firefox and Discord handle animated AVIF fine (also plays with video players like MPV, which can view frame by frame).

**Comment by QueenieArtemisia** [comment]
Yes, animated .AVIF support would be an absolute godsend. I will be keeping an eye on the project and hoping. 

Thank you for your hard work
